src/storage.ts

```diff
@@ -223,20 +220,18 @@ export class Storage {
 			// Ignore all errors so we can catch a 404!
 			validateStatus: () => true,
 		});
-		this.output.appendLine(`Got status code ${resp.status}`);
+		this.output.info(`Got status code ${resp.status}`);
 
 		switch (resp.status) {
 			case 200: {
 				const rawContentLength = resp.headers["content-length"];
 				const contentLength = Number.parseInt(rawContentLength);
 				if (Number.isNaN(contentLength)) {
-					this.output.appendLine(
+					this.output.error(
 						`Got invalid or missing content length: ${rawContentLength}`,
```

Comment:
```suggestion
						"Got invalid or missing content length",
						rawContentLength,
```

The methods on `LogOutputChannel` all take `...args: any[]`, so we can take advantage of that instead of continuing to use string interpolation

Reply:
Should I replace all messages that have a dynamic value at the end and remove the trailing `:` from the message? 

The only issue here is that we cannot guarantee that VS Code is going to log those in the same way in future versions. Is it simply a `message + args.join(" ")`?

Reply:
It seems this is how VS Code formats the args, primitives are kept as is, objects are stringified and errors made into human readable errors:

```ts
function format(args: any, verbose: boolean = false): string {
	let result = '';

	for (let i = 0; i < args.length; i++) {
		let a = args[i];

		if (a instanceof Error) {
			a = toErrorMessage(a, verbose);
		}

		if (typeof a === 'object') {
			try {
				a = JSON.stringify(a);
			} catch (e) { }
		}

		result += (i > 0 ? ' ' : '') + a;
	}

	return result;
}
```

Reply:
that's really seems like the "right" way to do it, I'm pretty convinced they won't change it majorly